code/datums/mind.dm

```diff
@@ -696,13 +698,27 @@
 		to_chat(current, "<B>[objective.objective_name] #[obj_count]</B>: [objective.explanation_text]")
 		obj_count++
 
-/datum/mind/proc/find_syndicate_uplink()
+/datum/mind/proc/find_syndicate_uplink(check_unlocked)
 	var/list/L = current.GetAllContents()
 	for (var/i in L)
 		var/atom/movable/I = i
-		. = I.GetComponent(/datum/component/uplink)
-		if(.)
-			break
+		var/datum/component/uplink/ret = I.GetComponent(/datum/component/uplink)
```

Comment:
Better variable name.

Reply:
And not "return_value" or anything like that, simply just `found_uplink` is fine.